client/src/helper/handlePostAdminLogin.js

```diff
@@ -0,0 +1,18 @@
+import Axios from "axios";
+
+/**
+ * @method handlePostAdminLogin send login details to authenticate
+ * @param {email: String, password: String}  admin_login
+ * @returns {String} - a response message if the login was successful or not
+ */
+
+const handlePostAdminLogin = async (admin_login) => {
+  try {
```

Comment:
Tiny general remark. When doing a try catch, you're potentially losing a lot of information when you have errors. If for example something in these two lines throw any kind of exception, the catch block will be triggered and the exception (will all it's valuable information) will be lost :)